packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const CODE = 'LESMIS_gross_enrolment_rate';
+
+const REPORT_CONFIG = {
+  fetch: {
+    dataElements: [
+      'ger_district_pe_t',
+      'ger_district_pe_f',
+      'ger_district_pe_m',
+      'ger_district_lse_t',
+      'ger_district_lse_f',
+      'ger_district_lse_m',
+      'ger_district_use_t',
+      'ger_district_use_f',
+      'ger_district_use_m',
+    ],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+        config: {
+          dataSourceEntityType: 'sub_district',
+        },
+      },
+    ],
+  },
+  transform: [
+    {
+      transform: 'select',
+      "'name'":
+        "translate($row.dataElement, { ger_district_pe_t: 'Primary', ger_district_pe_f: 'Primary', ger_district_pe_m: 'Primary', ger_district_lse_t: 'Lower Secondary', ger_district_lse_f: 'Lower Secondary', ger_district_lse_m: 'Lower Secondary', ger_district_use_t: 'Upper Secondary', ger_district_use_f: 'Upper Secondary', ger_district_use_m: 'Upper Secondary' })",
+      '...': '*',
+    },
+    'keyValueByDataElementName',
+    {
+      transform: 'aggregate',
+      name: 'group',
+      '...': 'last',
+    },
+    {
+      transform: 'select',
+      "'Male'": 'sum([$row.ger_district_pe_m, $row.ger_district_lse_m, $row.ger_district_use_m])',
+      "'Female'": 'sum([$row.ger_district_pe_f, $row.ger_district_lse_f, $row.ger_district_use_f])',
+      "'Total'": 'sum([$row.ger_district_pe_t, $row.ger_district_lse_t, $row.ger_district_use_t])',
+      '...': ['name'],
+    },
+    {
+      transform: 'select',
+      "'GPI'": '$row.Female/$row.Male',
+      '...': '*',
+    },
+    {
+      transform: 'select',
+      "'sort_order'":
+        "translate($row.name, { 'Primary': 1', 'Lower Secondary': '2', 'Upper Secondary': '3' })",
+      '...': '*',
+    },
+    {
+      transform: 'sort',
+      by: '$row.sort_order',
+    },
```

Comment:
Would be a nice tweak to drop sort_order back off afterwards through an extra select, just so the front end isn't getting clutter